var/spack/repos/builtin/packages/hip/package.py

```diff
@@ -60,6 +62,7 @@ class Hip(CMakePackage):
 
     # Add hip-amd sources thru the below
     for d_version, d_shasum in [
+        ('5.1.0', '77984854bfe00f938353fe4c7604d09967eaf5c609d05f1e6423d3c3dea86e61'),
```

Comment:
This is the only checksum I couldn't confirm using `spack checksum`.  What am I missing?